src/main/java/com/radixdlt/crypto/hdwallet/MnemonicException.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.crypto.hdwallet;
+
+public class MnemonicException extends Exception {
+	public MnemonicException() {
+		super();
+	}
+
+	public MnemonicException(Throwable arg0) {
+		super(arg0);
+	}
+
+	public MnemonicException(String arg0) {
+		super(arg0);
+	}
+
+	public MnemonicException(String arg0, Throwable arg1) {
```

Comment:
Arguments should have better names than arg0, arg1.

Reply:
sure, got it, this is inherited copy paste technical debt from `CryptoException`, will update there and in `HDPathException` as well